Neos.ContentRepository.BehavioralTests/Tests/Behavior/Bootstrap/FeatureContext.php

```diff
@@ -59,8 +59,7 @@ public function __construct()
     {
         self::bootstrapFlow();
 
-        $this->dbalClient = $this->getObject(DbalClientInterface::class);
-        $this->setupCRTestSuiteTrait();
+        $this->dbal = $this->getObject(Connection::class);
```

Comment:
why do we use the `Connection` here directly instead of the outside world `DbalClientInterface` provider? ... in the end its the same ... must be i guess?